composer.json

```diff
@@ -17,15 +17,15 @@
     }
   ],
   "require": {
-    "php": "^7.4|^8.0",
+    "php": "^7.4|^8.0|^8.1|^8.2|^8.3",
     "ext-json": "*",
     "juststeveking/http-auth-strategies": "^1.1",
     "juststeveking/http-slim": "^1.2",
     "juststeveking/php-sdk": "^1.1",
     "juststeveking/uri-builder": "^1.1",
     "nyholm/psr7": "^1.3",
     "php-di/php-di": "^6.3",
-    "symfony/http-client": "^5.1"
+    "symfony/http-client": "^7.0"
```

Comment:
6 has PHP 8 support, ironically this would be a good place to use | to allow versions `^5.1|^6.0|^7.0`